CHANGELOG.asciidoc

```diff
@@ -106,7 +106,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Packetbeat*
 
-- Add support for mongodb opcode 2013 (OP_MSG). {issue}6191[6191] {pull}8594[8594]
```

Comment:
You've lost your changes to CHANGELOG.asciidoc while merging.

Note that now we use `CHANGELOG.next.asciidoc` for unreleased functionality, so add it there.